app/gui/view/src/project.rs

```diff
@@ -768,6 +768,7 @@ impl application::View for View {
             (Press, "", "cmd s", "save_project_snapshot"),
             (Press, "", "cmd z", "undo"),
             (Press, "", "cmd y", "redo"),
+            (Press, "", "cmd shift z", "redo"),
```

Comment:
This change is already on develop, isn't it?

Reply:
yes just git screw up